src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -0,0 +1,33 @@
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+import { AccordionComponent } from "../../../../../Shared/Accordion.component";
+import { BarChartResultsComponent } from "./BarChartResults.component";
+
+const dataStyle = css`
+    max-width: fit-content;
+    margin: auto;
+    box-sizing: inherit;
+`;
+
+/**
+ * @desc Returns accordion to display the results in a chart
+ * @param props - data to display a graph
+ * @returns {JSX.Element} - accordion with the result chart
+ */
+export function QueryPageContentBarChartResultsComponent(props: {

```

Comment:
For a future refactoring, the accordions and overall style of the results page could be encapsulated in a layout component.